### PR TITLE
Add CI w/ GHA to Build & PNPM Test

### PR DESCRIPTION
**Pull request by joshsc63**
This change is [Reviewable](https://reviewable.io/reviews/lightsofapollo/storytime/5)

### CODE REVIEW

File: .github/workflows/storytime.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+          
+      - name: Install PNPM Deps
+        run: |
+          cd ${{ env.BUILD_CONTEXT }}
+          pnpm install
+
+      - name: Lint
+        run: |
+          cd ${{ env.BUILD_CONTEXT }}
+         #pnpm lint
```
Comment (lightsofapollo) [inline .github/workflows/storytime.yaml:78]: Does this run?

**Review by lightsofapollo — COMMENTED** [review]
One comment about the linter otherwise LGTM! Thanks Josh

**Comment by joshsc63** [comment]
uncommented lint & is included now

File: .github/workflows/storytime.yaml (hunk 1 above)
Comment (joshsc63) [reply .github/workflows/storytime.yaml]: uncommented, runs good now

**Review by lightsofapollo — APPROVED** [review]

**Pull request merged**